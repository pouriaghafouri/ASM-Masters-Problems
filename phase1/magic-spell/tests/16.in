6 g8HOEAL_Qg8YVdjV2tgmGivY_lnqM2nRXSuqi6nY_kLSmSCd_IpFRIsQTD6tsg_fZrt7kHSiPr017s26YfyCNPiuhbNAgKF31YX_sEh8V7WSmOLaR_CtOINTIniOXLEMxqLPSSKDOuiZ6RZMOq5rXG9uTe6EQzfaNONuvnX7F3TAIH_lW2BIxz0eDVGghJgUmUdF5ZzxPiXmPWyyNZ69lTaODGTKeZtqGvTyjhVYnH91qwo2ii0t1Y5stN2xAjL4EtsLycwMIuCWOONWWAcLn83EIU161z0sQf1ehnONFdxjR83By6bJ14dwN2sCrfU7kL9rWM3qPpxumqVIjWf_Pgvp6LRkQBf0_dfVO8aqkyKuMs3Vd6SQba67MMfrakh8nkS1fRfpcNHM6zw9sbNrbImNhr5gB_dOjUDmyGByi8_dvtaLuN3vke9pv1vWYzx5iYgEtH2CC1DVjDwDdmWnqUBJIWsvkPpqBvUU0JjqzNaGdu7hGTsUATs9FIrNnH4O2LvQiF6Hh4bio8pSOIAeqQaTorwzYnaOWvt4YmB3fAbrxq8bYxdcadVosgLerLQd577SivLnVMFg0AfMXiOXjw_zBIDSjiVbeQTmaryU110Rq5saa4VjpUGOq_wzfeAkUguTyQCoGr3WjkUjbFqpmW3CToZZtm7atoTP5k1yzUiG22P4w3iJNa9vnYhG_bEqqn3kvUIVDQosG4l2UrywsxQDkX7hZAyeNqm78SPyw3R2W2TEhf0zCF0KqWRelcgXrqTo8xNrnri8i0Bp5qMuUB4_kKbwLhg2vZdsjOKw5QtdDUqv91PSpIPAgQWPV0GfOHvVl3efJwg_dxvbl_RAGtYK7HoRwSU8ofSz8VFEgLEj6XjeWYFpfsZb0cTlSCh7FZvDHY5NzyVsk2x5P0jUf7U37CcP2jLx64YDSSfeeYwmPReuGmbKiVCckERcNpxHjk_12e543ogH6kZzwZ9CHzFS1lSCzc9Im8wb_q42C_xxhjVC6R4Bfwg7GYw5YuC8rY9qb0gDXBZ5IHuEnl5rIaldWH6J1xEfiCvhqzIc0u5y1o0cySr6SA7B528TmBYs1gzgG5jvpdRdqGdOnuH3UDrOsouCPhUEoiKUcRcpTSt7lwKxPepwFF9XTsowA60MdxtdctrVaKPm4mHGqU3Vo1F8gRtGMgfNByODS5lQC00u_w0pgQ8uGArNeIgOMmqdI3EpXO4nBTHBfuOlaKDG_VgcqlQ1v723ZvgJ7_WySqXVLy6L8ze6hj8vsNlNHcDtwJ1sGKOm1y7zX38UpbNuiKPAlZaQ1E_kaYBHxeTlBPKnFH5SHIbPfPdqDd7rF4CGR3ab5TntvVGZsBGX7GzbiCsLEm09drEShCTak4tEPYq5oURipfjvIz4_JU8M9Are2aea2vCGiSAuBgCPkK_Q92PGLMfH_xL2vN0m9rRg9gzzl1bvy_av1P1Bo66oqHfLj5WfjCmhGLERzGbYFbivO7U32PggjjPrbAwk0IpwgTcEmcrRbx_QtRGiuKoDR4USrg0hMeDRVFi8HzMHUMmaqSiKpvd6oVNFZAKzCcfmGnhaKUHrvTrLyzjNUmI65izUG9i8bliFwpF69ckrTB1fY_QGjoBduYXZX3WWd2qzhTsdVCvC1lHPikkryJuRvfFf8o1_eewlXPmGeH66QAIYKQc7mxNHCfMxrAwvEREpvRTxlOs0dYLNDOJQajmAmXVEmfXPWp2g6KBelS4oFDZ9gwOeDYEQKmhVr2zcgog_YIc9p6kUw7SCstEUh6zRiEBzvYCAbGzXdBTqv39RYQhd8LXeHjTPNib76Bw68TS9iBpDrlheZmiWY30scH5PP6JKx3DsMlqUKDk1Op4DC_on2cF4JzHnsfVP9o7Kxlr7OzVqOOTfN5qD7VwDh3Qy9za5L6C8gR2SfM8ToOXZJcrDWO71EKoA9puJkWEqAv6F64ojH98dapHU1BLuaXT8bcEjX9xmrtSmk4vR2rT0EpIsOiMO3sJ3tdbf_zpoS5B2WUGLamzEz7WccwP5MnX5oZ8llnA1fZQ2IlCHwZ9VVTYXcCSONCJ2oGazRApXm5MVebpA1nkL66w6xbKaDg0SK0erosbymNioOvMCJVdCQxyby6_1_ZGJMJ0Y11maBspfOZFkJIKn56mq1lh08LzHjmv_bFaCXnFy_aIHGgUzb6P3eUQbvcICn1AovyQgLjEKjaePgYdhRR8VAMWTMslZiLaE9PKHYbe7_jJd5LlMrguPQgysyHeEhf8dUFIHFMBCT_GOKRnZVHEBL2R7yVBDYwEH0bbuNCWwCr8_6kXPRqdsfUoBCN2q_vVZwXg7o0qOfnM9vyNaMOqRvCfYf7berW1LG7FTUWwZ76YstAs5eGJz8OkcIjeX45xzZ3hJMCxIxk0PJfJLLi8JUgJspOeiGAGvqN22cozAxne4qLC2RzyBF5RTITZeguxw6ol8AJvVWzM_9dZPBkd4fSMNBBPHT_2OlcMKJ7ssu5sq6g3wr7quM05coKI8HxLRxkq7fGoxAGNuLFQ0z4jbUdb8LH5g4EXrOcnRIBd76OqGj4vGYCHHFHCedvk8Z7oA7zhC0iyVXMo4GHzt95YCznGCIQzuMLTG808Y6wGS6TJBp7Sm1DOpRjPmXcHwLp2JcZw6i0OeGkDWgj65KJhqQXBAZ8WzkN8kzrfIfRKL2au6jp1UW88BUy1GtK6Bk1LKiPfltPWiPgeXt3Gf0BGI0vdg59IdYgN6TSecu0i74kTncms3OWADeDJ8AeallNe2si4K8aG2kpcnBsdcbDFg4cdrebCnMGf2O9BK77zhjBuJTxLUYgYPgZ7i0wwz3BqExRduOCBV1TtHf5p19cFe0zkPWGeM7sfu_hOLHcvyVP3YJhOHigJ8GTKppM1j54D2aeCGgV21zUO60pA6u730OAe1a3_3WBSVFjqM2fApZckMrJHJFxyiYA7Wq7McxtV0C9Brd0G02PCgn9QQubXhCT1SPCS_VA0kwoBzd6n3KN_VLNA5Om_d6YUIpzG80tflHHJJAWB_yyS7ah0PuXQnJzTYZmUMFZVattJTc8QywxtuFg7X3N8B_OnY15xvRivaB2RC0xYjSgDkmy5da2C_ENWuFgNmo9mOY1fNl1U48ldhJ9iHYJEpjnJPukZIg9l4ZoFafo4ax5f61mCNT522hBEBLs8RqgIdtcdwp5wMXzIOL8qr0hj8GNwe4rThyrkRtlcIfwi3VOEuLUJz1SwxtQzk8irvzC_QL_mPvuFd89vGS55GKo133qc0ysiVSfAseM7z6BA1zWxeOpLnCzeuPghawpVdulwwXqTRQiFc20hFfQTFeXZS16SniDoAMIWwZEcCMGEEvL7yoP3tzP9oKQzSgNhSj1e8uex5K1w4AsrO6i5uX3GxH5cNGitP9xKtB7l_XH3kZjZUrTceKIzgBBR7J_JGuiXWc8TXCL8oUU9_A9b_Eyd5XTZuSx0cDMWDHEq9elmW5iTueSsJfvB3dqkUL_UcJFFf7Tda2nUWwDeAiw7oPyeROmymlhoUMSXGAYGsanbuOfTXzP9ccaTOmg1vnceXUZqiMWyM7z5LDNx0rv2stK4FOS0ZV2JCQZKpI6ZFtTfUvAJLVB1b9UGNCtzkk9MYVkatfZWyI1f1qPz9fp_b_DMzWbJ48kPQsP7yDSU9G00UCoSHEPGBhiY1hwScEwSXaPiCw2A3PpMgCsL45hsanf1uzJu33bN2QVtaLT3n7CuwS3pN8HNkKBC98W11YESBmaBXRraLSs7zkj_iOMqndRl9CaZnCGNOEdBltzUaRRxZ0x7C7lP82Z7rZVDqrf4Tgs2xRMvvsTkffgF_pk8g94wXOZa324M8kDt1cOmuwwyzA1zOawS7pdR10RURJtNTW6HZJRg4cCsBtPeraWonY3eNKYsge5Z0YtMwaRpagGzxkDNjnZw9RALol1upVgdIAN3ArfyyLYTUov3BjoxZN7cYXxbGBed1P4o7_LsIyarMwjbDyyr9usWfNVLdXM3CGgyDRQacOsP8zOBVbQTu8FzIn_MaLECeIYHnBIpeY2cmFB8FgOZcilKukjuTNWWjJsw8YJbJBdU4EPyJsmJzxgTFqcm1W8_sntpb3qKrstkW6JsoTZNe6vHjxT8gO6y2oN3e3AvkSE5Ncm1T_DX3Wsbga7mN2KEemuidUCnz6fc6q1OpszhdRgiRcut53V9cegfXIrlBwnwMea1uz8xfdFUgYb9QU7QZaWJHcU8y6ua8sZrPXOSYhz23y0GiWl5V7Dr9mmvhEteWhvy48fQcFH53xAagL51GxfCIsXPWfSGna4e88U8Cr2sMprS1jJwFOWc4JPOMujXva2q8JoxYdQzt5hhdOBGrmGjTlXvD6fW67c2Eoorp2RIWWNYzdtQq0X79JrBExnyrnnTzBaOtPkcAX0ZZhCc7p93YotqLEM0RZJdnHSEl0HJMuxABZCxbzp0NHdlaPaQBHTObyiknPTY9fly3MV4a9TNDUYByWgZrYBslJDxllijptHf5q93mQDPzqdWakKr6aHeH8BShH0w1uBTIxKUccwCrAlrIVGDUaFo957fBWzrd09MmItnKQNZdYdLGJbnKGCIyyL0h_ekXcU7IdjgRW5UH9sdFtqcXQIjbi9ihcr4d91Jb8PSSIBoERp98GbTjH2lQ0tV0IN3HBAGyeofKNsduHara__kG1vlOM7EiHuN8UivXuzu8PyplwGmvEuZsNa7o5yuBS7wAfRkzdEuS0J2ldbEHvtZ8rTjlgEKXA6mDKw1MYktOSkZVltpEKbB1TJbZcJLACV2bgSN505GFcunmLCQiCf9kO87OGId6q58xKIpAAV3DcgNMGsU6xRfaOaMiGQdWI9giFtG5byveC8O9ov4Lb8KO8lWDZZoxWtDqM7iOuEQVztTNMKmOHWrE5a5UXheIAHYbD7NVysFXLnAlWKXsvbVnbN7YUawhFjh8dUTBcloLzMWilIAEHiRIWMwDMQLhXQc1zVrz7DatcUBlqZPX8uvR6PjFuSKgxN5XvmlpNjIQ471jUE4PYovR1ElimWpJwhtRtF75OCKFxyMg2GTNSdrHFBPRk2oG_Fkq_gkMHUe4gR86kQH0RYwlob1y1c20HcgHgogN9iEcXz87eEXTqg5EhV19XQWtS0YZe2zaa4cXs_SvCCdSI6jQQ8MAOvStlDfptFpryrbOoT9OkcvR8DuMy6__oPrziwOz13QmhR1wAZ8IZroVUIufBtfQ7uceQRBHHgRNXH7ku4Rgk3Z4zi80AaDwuGy_kZER3jtPOpY8jEctwZlT8gJvg_PyFaxPX0uP9LrM1cI8GIo1wxJrDfMHfo5Imql9fDLbd3N23iXzQ_ncwUtY04u3fnysDHqG_0HbQh3IqQ54Nh692nUPecFhp1zQyOlxDSyiXpQc3L7FQ0BFalhcnKkCBJhZkqjN6GTSTxVL6PdLCEeBPlB0WLrkiw9sKf4FLMnt9624I5D_xHybQ02zygJEBFVaLOGjnRBmMsdjlGgIaDHRswfODNi2h32QEvYR_o1LE5SNyXkzpQdFbgiCTotZejEICrnCDdc86TIuFTTUyUox0u0JJivNrXjSewjFxjAt29NwRtdcarz0MowivRVKE2rGlAaJKB2BzCWdTZdTeAGON3W8JEHauXGFmFdU63jss5t9RuPTUlwudfrKIXL2JewTJyDCoMSFE9Dksi4atnuutK41uCS1HbHfx_IMJngcmJlEPeE6pXpIujwOLbCfd9lA7R_F4fIfNRIpTbl8WAGfH0Rg2gl5fusbasE3v_86QFlyFwwoW0srP7xGdIyg1Rf17JRpJPlohuNKQ7XzWeQAaa5bHshx7ml1W3esFnGMHgkka7JW9nk7nc7Ssdepoqd8ihpNuTm10waYqIKmTSh6SdO_gQos6qpdxEOQn1EakDYzavLTaQMSTnSY44fXjvZFXCi_sM8BeW0eeyVfexXVICIzuOmB7a6TCbRTOOjQx9TCuDFyYrjuSPTbq5Cm5w4xxId9jmPHiv7CYBZKQ64xKMw1FZazi2UDxXCH7ebez8FkytSboLzWlTKQHJcNyXgWHGsEIrI5oaeKtXLFv8pERZjlxmZU73FBwV54bBYnB0XTKvn6tCz8pHiKR5tLVYbgG78HvTUWJGEi0Qbihzfu6l5Kgnk1_l6Ff3aAMUmioOodtqlzNoTGzLfFXnuWynoCdo2N9bVkO8n7lwtYKat7LwAvJSg75tw8HoLDntL1qYWBheoPcFMO1b9zTduLuQIZ2gqfA14QPPhUSvzUZavP_scQtwqN_Wz23P5sGY8iBe2iyq0mqjZnB13Skgsk3R_Vw2a2N8_da1jyh_IvguJHlBmvIEDykCgGtgv7du8blhxqe5LkphPyJ1TeskACMG74MrYNMVPV0ba25J_hROFnClpSvNk8igZSxMu7u7Rk9P_3m__1LCbbNqT933497SQsr_mL72T3GTTQSQGqhHrUVaSL1LJYqodFmpgggX7LDPoiuVzBpQk9O1JFziqOuriEGmKL_GjAixJCf63tJbrn00So660om9SQkqqhWzI5Veu0_liImxT_lAlgtbsF_alhqzn_ZT4HXmwUxDpJ09GawRfQSxiPvrXLR8LDQC_AOtjaWwHKucIPULsz9AerSZ3yUBZz3YZFg8D1raAMa6oSHGfD6hUL6M7PaVb2G0wNVXBbV1LWUZBquGToNBZGaUibEj3a7mLR9bRU0rDJe4YyyGMa7zF8iN7KVZIR9iy5hcO5teEv7rUuVtsQQXNWAHwiv2Zr9klfmYaE2CZYRGhB0ygDkRpInJQGAFXyP7C0UAtJ2fxIMC7z0d29SeG6NkzbNkzpgZeYovwdBRJajEz7Fq5mvzf6JF7jNuyTiDGvW0wmH4mQxLK20Ddi2hbyK9Hms33yvyRfldrS7Cvta6iYyl6ns8J24dbj64HppmuBnJijaEAbzIXVRR88MEXFHZOCSiP8s9yFIGMG_bFwHsgo9bjKF6e_UQCAWexedVJJolemkKGQ2N2YMaxh6zgOeGdYLA2MizkuImESWaxLNoymmVrijyUl2X_AkZ_qnjKVweBfcYPPacZmWgs3CbctW_S6YRj_ZSUiUinW52y53meMsxEUwGbiFTbrzjql18tKgFtauR5m37Yjtq3OW2UpKVG84WiSTArX5KXooPzfMko3AeGk7nzE93z0MGIs6XfYv2bJFyOgJ0775BgZODsKvQA7l8pgSs4cj3MOrnSYdPU6f1URr_rNPRH0MU4tbVujO6V3gDQhfxcinUZDSffuWKkvsbPrX9yy2jo6K4dPqdXCWL5DR8VAF5WV5ycPwAclHqeghhVvjHX5gPvXKfkf9601t0FNqFWXT14Y8Mieh47nGr8gvhlrd9huZK8ccRZVHQIDqOHxFEKbTFhbMqsNox6a51c7H2PeFmHVZbftFNumhznSPEs2Zl2Ub5OJUtLoDf8rgaJKO3WTq9y5ARWp2O92IMyp39BwdwNhxlRacCRszfkZW5dLI_CdK0qCWRYXdz5AILyKbe0Ah9o33sCyr2zZSPrBveowDiWbRhJSkxhrtuklMKH1B6QgI5I39UnA09BGe8oostFLNPWmmqbMlPSTJnKF7X5UTtqXrCaHTGiwi2GviF7rjMaQYIlUsfEazSYOjWk2rqlAiQTokNFuoD8b9iUqolrL1co8YwXdN9DVOj7WWBd83b81jPhvYw6OyjLjExmgtNZ5WVRFjUBbVw13a6nWrjzNq8WUs70MJMEtwkWG3mFLGtDGoRrFYOfbL0W2WJDsl7JFfv98FEGbX_F_OVEByIC8xlZgVpy4l6woqvjTXQUHNnGq89PvGgq1qe79ts2DnnPDGZksC40egtubr9xVcNLsQHq70hzatcCX1MfsD3wIlOJBM6lOHV7kpkgcqEdJGFvweyMHo6eAJOZiH8WbQQl61s8eWaLrD6LHskeGfg4OSQWpOFqtjAm8SkbBkMQMFqiVAkrDpig7Z0lAsARz04wFdv4l7JXAX4kmbzNqIRnwGwUWUAlHtFbw15GVDqkphtzjSbznSKHl5q6ORgl8LkszfYkGbsUjLgsrgPEWnb6hr1V6544Fej5h7cL8use6xwmDbOmmPht56bY06Pt8YlcSnLOH4SAqcKRdo1n17GT2FHSAj9v8jxNw7q107pd9ZS_cIm2EQKukgbJo_4lgpWAlaq97Fb3sR2sobjRRUa1Y_zb7rkcE7CN7QL1jKTBqKTCJ3gpKgdvdBv_QF_jzzWvfwjm42NjztJ8jQkTWocYPxXu2VBrjmKmGUyzLaG9se6BUgiEskp5H_owUNL2Xldq3CNBAizQkth3zpuRzJL4IXpqxZhj_kz0Kbq9hOOr7Tjt8B8u_GlGqBWNn1Wmai_JhnFo1gDV0LdVbaf_EqFUPow2efbclaLsmdEck8W8GZSHYVEpb9_OmvEqzFrKE01O4uQbpz9TmOo8zSmyRlb1FDf47wBAlrdcKRdXe_GqMUnaBzyfLx5fZ_7VtxkCPlFo0Ga4EEhfnudPUzSsUMvIJrs2OAsqJVEzqDqS7yVkQYXxmCP5e_CNqVD3iViSFLeTdsz8Ojfu62Rfsu_wtC7IkxAshG8KgbrhuQf6XkpSa5WsoVMHkGcs4AJ97GJcGYiYBnTnv74v1PLcxkHIRKYK_wI43e5w2cjsp0EKWv4K8CMuNj0rTO1Q_yK0DElsEsJTiaq4vhCsIdKkmAB3dpHaLe1dG_u_s3Qy14plbRBJVaR5yfWBusC5uq7YnoXFeA15tgduXE1FCFA1Kjr2zRVT5PGi3qL5QCZ9I0rsthYTLlJiuZaTDTzyv5EnkefYE2W_Ras8ifQRqpZKdXqGDP2YHt9PydC05l0LmhIsko9ABX8EIl8a0YZwgXaC1CqTNfsZk0gubcSA0N2vY0vNLj7P77fVwvD7AVUKJY2I1J9Ol099NtJlDF1xz6fVqIPOrylYmcvcLrQUhOSS5o4xTUShNX03tEEI3PvcROer6SaatQQn4JJKqqP3bCTu6m0W1kYHW0WPHUP0xtbo2KYiYA9X2SeW44HTbtnYg_CORfBcyAqp_cvWCuIs_Nd1GIJI1Vt8SVJyY8ylyPzvRSeg_Ny9nBYR_umYeFTWoribnGlLkK4Zp94pJrlWR7COrPztjifxxlxKRIi0fcMs9FFHVRszLSc2wzlEHoZ3zlAdRJ3UJFcF8HbSlrgX9gMtEVX99J1vTrzzYrhuUjZQQYx2eCMaGnt_7f75PXqvbNUX21eK81pMOKBQbcRGn_DjdzcQkqaj3S7UH9tE0GdBgEhfGWLUTc1Wz3zJizRafNTMJ_dyGh0ML5RfOqXwqNiPEHmM53L_DrJbqKyWPlvofarTqdcGOiihQsSIkqFLHbNXJ9Hoj2Am2R5fS5JtdSBR_goiwRT3StNqpjCy_2KZGC3lxBFAg4ge_skv91lQiW4x6wW3lvPQWH1i7vGaoKen2oGYdPDlBuIuOrk0LZFwuvCqP8q3HtewFIjInKRM4p6IGfvg2YAlg2BVZQOt7QPBnY8AxObp15Vv_fz230a9RASEdtX_9CylnvKJ8JYWCGgBLGqBvpx_PeDSxqRtQcD3waBtJmeaQlAqPOP_3c7HeAmAQ3Tt6jlrjLIRVKRBTePxRuuJurewRqWuhDNaMWpUuXAfueGdjiAYATwU_yeON06UqGS1r7KMR_PabidkqC8RksyhOCTqqO_GhRwyL48r1Ms3Sta9V7N3oa80BQgRrdmyTHVtyQLpq1h8jh63dGTpHRf8ulMKoY65sOoRuX4KNaFUhBKj7tyB8BwpK8ZXUTR_w5OOPHlpHR_NhHUb0fBVHXagTX3ADKz0CdBhKNwfttQy1zxOEYx9Jvfrh6RLFdy6gZaPBwUSOxfCWDdnqAmXVrb1mQzQUXJYJJC93jQGGW6rm_FCzRni6o9f5xTOhrzQ02MRasmHbiWnhoNE30MWcI275II_Y7QNWptXH
