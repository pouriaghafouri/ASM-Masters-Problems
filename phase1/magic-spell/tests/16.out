According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because_bees_dont_care_what_humans_think_is_impossible__Yellow_black_Yellow_black_Yellow_black_Yellow_black__Ooh_black_and_yellow__Lets_shake_it_up_a_little__Barry_Breakfast_is_ready__Coming____According_to_all_known_laws_of_aviation_there_is_no_way_a_bee_should_be_able_to_fly__Its_wings_are_too_small_to_get_its_fat_little_body_off_the_ground__The_bee_of_course_flies_anyway_because
